web_responsive/static/src/legacy/xml/form_buttons.xml

```diff
@@ -9,9 +9,17 @@
 -->
 <templates id="form_view" xml:space="preserve">
     <!-- Template for buttons that display only the icon in xs -->
-    <t t-name="web_responsive.icon_button" owl="1">
+    <t t-name="web_responsive.icon_button_create" owl="1">
         <i t-attf-class="fa fa-#{icon}" t-att-title="label" />
-        <span class="d-none d-sm-inline" t-esc="label" />
+        <span class="d-none d-sm-inline"> Create</span>
```

Comment:
Can you please change this to New as the newer update of odoo 16 has it as New instead of Create?

Reply:
Done! thanks for the review